taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/tasklistener/TaskanaTaskListener.java

```diff
@@ -87,13 +88,14 @@ public void notify(DelegateTask delegateTask) {
           break;
       }
 
-    } catch (Exception e) {
-      LOGGER.warn("Caught Exception while trying to process a delegate task", e);
+    } catch (SQLException | JsonProcessingException e) {
```

Comment:
Wouldn't it be better to stick to the general Exception in case some runtime Exception or whatever else occur?